vm-setup/roles/common/defaults/main.yml

```diff
@@ -58,6 +59,7 @@ external_network:
     forward_mode: "{% if manage_baremetal == 'y' %}nat{% else %}bridge{% endif %}"
     address_v4: "{{ baremetal_network_cidr_v4|nthhost(1)|default('', true) }}"
     netmask_v4: "{{ baremetal_network_cidr_v4|ipaddr('netmask') }}"
+    dhcp_enabled: "{{ virsh_dhcp_enabled }}"
```

Comment:
We need to either default this in the j2 template, or add the new key to https://github.com/metal3-io/metal3-dev-env/blob/master/vm-setup/roles/common/tasks/extra_networks_tasks.yml